build.zig.zon

```diff
@@ -9,7 +9,7 @@
         },
         .godot_cpp = .{
             .url = "https://github.com/godotengine/godot-cpp/archive/refs/heads/4.4.tar.gz",
-            .hash = "N-V-__8AAJvIeABgT5UIxF6GOJediw0qZYqfwvrgwovjmevl",
+            .hash = "N-V-__8AANvNeAD_SV1i3OCIDHR74eH7M2BTkpa3UyVKTHph",
```

Comment:
I should call out that I updated this hash; you should verify that you get the same so that I know I am not getting MITM'd